guide/install.md

```diff
@@ -86,6 +85,10 @@ There are several reasons this is needed:
      downloading it), it stays around until the file is closed. Moonfire NVR
      currently doesn't account for this.
 
+The sample files directory can be set or changed using:
```

Comment:
This is misleading. Currently, on trunk, you need to pass in the same `--sample-file-dir` arg to every invocation of both config and run. The only time you should change the value is if you've manually moved the entire directory contents between invocations.

If you ever specify the wrong path, it can get confused—some files written in one dir, some in the other, some files that it thinks are deleted but really aren't. And the best process for recovering is basically then copying/moving everything from one dir to the other (assuming you have disk space) and deleting all the files "moonfire-nvr config" says shouldn't exist. It's awkward and undocumented.

I think it's best to just not mess with the default storage location unless you have a good reason. You can set up a symlink to another mount point.

Some of the `new-schema` branch changes are intended to make this safer. The path is stored in the database (paths, actually; you can have multiple dirs to support multiple hard drives/volumes), and some metadata is stored in a file in this directory, and rather than getting confused if you screw up, it will complain if the metadata is absent or doesn't match the database. I still need to work out a UI for fixing it if you've moved the directory or made some other mistake, but the safety checks stop it from making the mess bigger.